### DISCUSSION TITLE
Pipeline 2 v1.14.16 released

### DISCUSSION THREAD

**Discussion by bertfrees** [daisy/pipeline#739] [Announcements]
Dear all,

Version 1.14.16 of DAISY Pipeline 2 is now available for download! The following alternative packages are proposed:

- for Windows users: [the desktop app installer](https://github.com/daisy/pipeline-ui/releases/download/1.2.7-RC3/daisy-pipeline-2023-setup-1.2.7-RC3.exe) (exe file, 138 MB)
- for Mac OS X users: [the desktop app installer](https://github.com/daisy/pipeline-ui/releases/download/1.2.7-RC3/daisy-pipeline-2023-setup-1.2.7-RC3.dmg) (dmg file, 182 MB)
- for Debian/Ubuntu users: [the Debian package](https://github.com/daisy/pipeline-assembly/releases/download/v1.14.16/pipeline2-1.14.16_debian.deb) (73 MB)
- for other Linux users: [the zip package](https://github.com/daisy/pipeline-assembly/releases/download/v1.14.16/pipeline2-1.14.16_linux.zip) (80 MB)

This release features support for Microsoft Azure Cognitive Speech Services, improvements to the DAISY to MegaVoice multi-level and DAISY 2.02 unscrambler scripts, the ability to select preferred TTS voices in the graphical user interface, and various other improvements and bug fixes. Please refer to the [full release notes](https://github.com/daisy/pipeline-assembly/blob/master/NEWS.md#changes-in-release-v11416).

User documentation is available on the [project’s web site](http://daisy.github.io/pipeline/).

If you have any problems or identify some bugs, please report them to the Pipeline team through the [forum](https://github.com/daisy/pipeline/discussions) or the [issue tracker](https://github.com/daisy/pipeline/issues), or contact us at [daisy-pipeline@mail.daisy.org](mailto:daisy-pipeline@mail.daisy.org).

As usual, feedback is warmly welcome!

The Pipeline team

**Comment by bertfrees** [comment]
@torchtrust asked:

> I am very interested in The Microsoft Azure Cognitive Speech Services TTS production, but it looks like a big learning curve for me to jump into that with the configuration file. I presume one of the team has some test files/configuration etc. obviously the key would need removing, but it would be a huge jump start to have the details needed to get a test set up. Is this possible please?

As a matter of fact, as of this version, the configuration file option is hidden from the graphical user interface. Instead, there is a new settings dialog window to select preferred TTS voices, and there are also settings to set your Azure API credentials. If you want to make it work without the graphical user interface, I suggest you first do the configuration with the GUI, after which you should be able to find the XML configuration file generated by the GUI. Let me know if that works for you.

**Reply by torchtrust** [reply]
Installed DAISY Pipeline 2023 v1.14.16 on windows 11 Pro 64 bit
ran a job: DTBook to DAISY 3
Filled in GUI prompts
Error
Input not found: file does not exist: C:\Users\PaulW\AppData\Roaming\pipeline-ui\ttsConfig.xml

Looked in Settings, TTS engine and filled in Azure credentials and connected Azure. Selected Azure voice in Voices menu.

This then created the ttsConfig.xml file.

Are there any Aural CSS example files?
Thanks
Paul

**Reply by bertfrees** [reply]
The "Input not found" error was fixed in https://github.com/daisy/pipeline-ui/issues/173. We'll try to get out a new release as soon as possible. In the mean time, you can test with https://github.com/daisy/pipeline-ui/suites/18683660142/artifacts/1087237309 (but note that this binary hasn't been code signed so it may cause a security warning.)

**Reply by bertfrees** [reply]
I don't know of any aural CSS example files. Perhaps someone else, who reads this, knows?

The [user guide](http://daisy.github.io/pipeline/Get-Help/User-Guide/Text-To-Speech/) should get you started on creating your own. And of course we can help you.
